backend/src/packages/journal-entries/journal-entry.controller.ts

```diff
@@ -109,23 +178,135 @@ class JournalEntryController extends BaseController {
    *    get:
    *      description: Get all a journal entries
    *      responses:
+   *        201:
+   *          description: Successful operation
+   *          content:
+   *            application/json:
+   *              schema:
+   *                type: object
+   *                properties:
+   *                  journalEntry:
+   *                    $ref: '#/components/schemas/Journal Entry'
```

Comment:
Does the response actually look like this? If I'm not mistaken it is something like `{ items: [{ ... }] }`

Reply:
You are right, thanks for noticing.